man/common/authorizations.md

```diff
@@ -29,4 +29,23 @@ HMAC tickets can be presented as hex escaped passwords.
 
 When using a policy session to authorize the use of an object, prefix the option argument
 with the *session* keyword.  Then indicate a path to a session file that was created
-with tpm2_startauthsession(1).
+with tpm2_startauthsession(1). Optionally, if the session requires an auth value to be
+sent with the session handle (eg policy secret), then append a + and a string as described
```

Comment:
eg policypassword

Reply:
Darn.. always switch those two!

Reply:
@williamcroberts this PR is definitely much better way of handling this